inception-project/inception
# #2154 - Use Fontawesome for left sidebar on annotation page

**What's in the PR**
- Switch to Fontawesome icons for the sidebar
- Remove icon files
- Remove icon file copyright infos from the NOTICE.txt files or remove the entire NOTICE.txt file if nothing else is in there
- Adapt the dashboard sidebar styles so they can be used in the annotation page sidebar as well

**How to test manually**
* Check the dashboards
* Check the sidebar on the annotation page

**Automatic testing**

**Documentation**
